Makefile

```diff
@@ -37,16 +37,42 @@ clean:
 	$(MAKE) -C man clean
 	$(MAKE) -C src clean
 
+INSTALL=install
+INSTALL_PROGRAM=$(INSTALL)
```

Comment:
maybe a  755?   maybe that's defined by default, and can be skipped, but if that's what we mean, it's a nice declaration of intent that makes this easier to understand.

Reply:
I would go after examples from some common projects. Any suggestions as to what those might be?